packages/lesmis/src/components/MiniMap.js

```diff
@@ -24,14 +25,38 @@ const Map = styled(MapContainer)`
   height: auto;
 `;
 
+const BasicPolygon = styled(Polygon)`
+  fill: ${RED};
+  fill-opacity: 0.3;
+  stroke: ${RED};
+`;
+
+/* eslint-disable react/prop-types */
+const CountryMask = ({ countryData }) => {
+  return <InversePolygonMask region={countryData?.region} />;
+};
+
+const RegionPolygon = ({ entityData }) => {
+  if (!entityData?.region) return null;
+  if (entityData?.type === 'country') return null; // country is fine without, as it has the mask
+
+  return <BasicPolygon positions={entityData?.region} interactive={false} />;
+};
+
+const PointMarker = ({ entityData }) =>
+  entityData?.point && <IconMarker coordinates={entityData?.point} color={RED} scale={1.5} />;
```

Comment:
I'm not sure if you know, but it is possible to scale the icons on the map using css by targeting the mapbox svg and using `transform: scale(10)`. There is also a `transform-origin` property to let you control the point of scaling. It still may not work for all the scaling scenarios you added to ui-components but if it was just here it should work.

Reply:
Oh right! I did not know that. I figure given that the `scale` prop exists on `IconMarker`, we should make it work - but perhaps even within ui-components I could simplify how it's working. Might take a look at that on Monday.